README.md

```diff
@@ -88,7 +88,7 @@ We always appreciate contributions from the community. To make changes to the pr
 
 All pull requests with new features or bug fixes that affect how animations render should include snapshot test cases that validate the included changes. 
   - To add a new sample animation to the snapshot testing suite, you can add the `.json` file to `Tests/Samples`. Re-run the snapshot tests to generate the new snapshot image files.
-  - To update existing snapshots after making changes, you can set `isRecording = true` in `SnapshotTests.swift` and then re-run the snapshot tests.
+  - To update existing snapshots after making changes, you can set `AssertSnapshot.isRecording = true` in `SnapshotTests.swift` and then re-run the snapshot tests.
```

Comment:
What is this change for? I don't see `AssertSnapshot` defined anywhere

Reply:
Oooooups my bad, bad paste

You should read `SnapshotTesting.isRecording = true`

Took me some minutes to figure out where to set this property to `true`

Reply:
I usually just add `isRecording = true` as the first line of `SnapshotTests.testMainThreadRenderingEngine` or `SnapshotTests.testCoreAnimationRenderingEngine`, like this:

```swift
  func testCoreAnimationRenderingEngine() async throws {
    isRecording = true
    try await compareSampleSnapshots(configuration: LottieConfiguration(renderingEngine: .coreAnimation))
  }
```

Reply:
We could potentially commit `isRecording = false` here so it's more obvious where you're supposed to set it to `true`

Reply:
Done